alarm/utils.py

```diff
@@ -33,3 +30,128 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def refresh_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in Threshold.objects.filter(trade_pair=trade_pair):
+        if ThresholdBrake.objects.filter(threshold=threshold).exists():
+            continue
+    for threshold in thresholds:
+        try:
+            latest_threshold_brake = ThresholdBrake.objects.filter(threshold__trade_pair=trade_pair).latest(
+                'happened_at')
+            if latest_threshold_brake:
```

Comment:
So if a forst threshold had a threshold, we will not generate brakes for another thresholds at all?

Reply:
Changed. Now we keep all ThresholdBreak for trade pairs